server_admin/topics/users/user-registration.adoc

```diff
@@ -1,30 +1,16 @@
-[[_user-registration]]
+[id="assembly-user-registration_{context}"]
+= User Registration
 
-=== User Registration
+[role="_abstract"]
+You can use {project_name} as a third-party authorization server to manage application users. Administrators can add users or users can register themselves.
```

Comment:
By the way, did you intentionally leave out the Captcha note? I agree it does complicate things.

Reply:
Yeah, I don't think it belongs in that module. I  thought it best to focus on one topic. reCaptcha is looking at spam and web traffic. I think this should just focus on user registration.